app/views/puzzle/dashboard.scala

```diff
@@ -157,26 +157,26 @@ object dashboard {
     div(cls := s"${baseClass}__metrics")(
       div(cls := s"$metricClass $metricClass--played")(
         strong(results.nb.localize),
-        span("played")
+        span(trans.puzzle.played())
       ),
       div(cls := s"$metricClass $metricClass--perf")(
         strong(results.performance, results.unclear ?? "?"),
-        span("performance")
+        span(trans.performance())
       ),
       div(
         cls := s"$metricClass $metricClass--win",
         style := s"--first:${results.firstWinPercent}%;--win:${results.winPercent}%"
       )(
         strong(s"${results.winPercent}%"),
-        span("solved")
+        span(trans.puzzle.solved())
```

Comment:
this should use a translation argument like https://github.com/ornicar/lila/blob/c2c39a5743ea9d96bfbdc939ebf792c768816292/translation/source/puzzle.xml#L30

Reply:
resolved in cf6a1d9ee65be1049b48e4cf0ea863e8f08d6e7a